embed_video/backends.py

```diff
@@ -322,6 +333,22 @@ def get_code(self):
 
         return code
 
+    def get_thumbnail_url(self):
+        """
+        Returns thumbnail URL folded from :py:data:`pattern_thumbnail_url` and
+        parsed code.
+
+        :rtype: str
+        """
+        for resolution in self.resolutions:
+            temp_thumbnail_url = self.pattern_thumbnail_url.format(
+                code=self.code, protocol=self.protocol, resolution=resolution)
+            if checkUrl(temp_thumbnail_url):
+                return temp_thumbnail_url
+                break
+        return self.pattern_thumbnail_url.format(
```

Comment:
The resolution `hqdefault` is already checked in the for-cycle. If it is working this line will not be reached, if it is not working it would be returned `None`.